build-tools/packages/build-tools/src/fluidBuild/options.ts

```diff
@@ -203,29 +199,20 @@ export function parseOptions(argv: string[]) {
 			continue;
 		}
 
-		if (arg === "--services") {
-			options.services = true;
-			continue;
-		}
-
 		if (arg === "--all") {
 			options.all = true;
 			continue;
 		}
 
-		if (arg === "--azure") {
-			options.azure = true;
-			continue;
-		}
-
-		if (arg === "--buildTools") {
-			options.buildTools = true;
-			continue;
-		}
-
-		if (arg === "--server") {
-			options.server = true;
-			continue;
+		if (arg === "-g" || arg === "--group") {
+			if (i !== process.argv.length - 1) {
+				options.groups.push(process.argv[++i]);
+				setBuild(true);
+				continue;
+			}
+			errorLog("Missing argument for --group");
+			error = true;
+			break;
```

Comment:
```suggestion
		if (arg === "-g" || arg === "--releaseGroup") {
			if (i !== process.argv.length - 1) {
				options.groups.push(process.argv[++i]);
				setBuild(true);
				continue;
			}
			errorLog("Missing argument for --releaseGroup");
			error = true;
			break;
```

To match all the flub commands.

Reply:
Changed in the latest commit.